modules/mod-pcm/ImportPCM.cpp

```diff
@@ -439,10 +438,8 @@ void PCMImportFileHandle::Import(ImportProgressListener &progressListener,
    // To begin with, only trust the Muse Hub, with whom we collaborate and can
    // ensure they comply. In the future we may extend this list.
    const std::vector<std::string> trustedDistributors { "Muse Hub" };
-   if (
-      const auto acidTags =
-         LibImportExport::GetAcidizerTags(*mFile, trustedDistributors))
-      tags->SetTag(TAG_ACID, LibImportExport::AcidizerTagsToString(*acidTags));
+   if (const auto acidTags = LibImportExport::GetAcidizerTags(*mFile, trustedDistributors))
+      outAcidTags.emplace(*acidTags);
```

Comment:
So this is the change: instead of being written into project-owned tags structure, ACID tag is exclusively written into new parameter.
But now ACID tag never gets written into project tags? Is that desired change?

Reply:
Yes, it was wrong to create `TAG_ACID` and use it for clip-dependent metadata.
> now ACID tag never gets written into project tags?

They simply don't exist anymore.